### DISCUSSION TITLE
Does the system support RAG in API format, such as integrating fast-gpt into openai-webui?

### DISCUSSION THREAD

**Discussion by Liyu910228** [open-webui/pipelines#207] [Ideas]

**Comment by justinh-rahb** [comment]
Could you elaborate? Issues should not be opened only with a title.

**Reply by silenceroom** [reply]
I guess the user is asking if the files can be uploaded to 3rd party knowledge base ssytem when the API integration hooked with other systems such as Dify/VoiceFlow/FastGPT etc. 

Curreently the RAG in OpenWebUI is not strong enough however other systems may already laydown a robust foundation which could do a better job. That by said, if the API could allowing users upload a file via OpenWebUI but actually went to the integrated systems that'll be much better to deliver a quality result. 

@Liyu910228  corret me if I'm wrong.
